src/service/resources-impl.js

```diff
@@ -1628,16 +1655,15 @@ export class Resource {
   /**
    * Requests the resource's element to be built. See {@link AmpElement.build}
    * for details.
-   * @param {boolean} force
    * @return {boolean}
```

Comment:
You saying we want to drop the return value? It seems to be still useful in this context, but maybe not in the `ElementProto.build` one.